tests/test_selection.py

```diff
@@ -21,15 +21,20 @@
 # #############################################################################
 
 import ast
+import sys
 
 import numpy as np
 import pytest
+from pytest import MonkeyPatch
 
 import mdtraj
 from mdtraj.core.selection import parse_selection
 from mdtraj.testing import eq
 
 
+use_astunparse = False
```

Comment:
Might as well only consider the case in which this is `False` and avoid branching out